designer/client/src/components/modals/ValidationLabels.tsx

```diff
@@ -5,20 +5,18 @@ import { Validator, withoutDuplications } from "../graph/node-modal/editors/Vali
 type Props = {
     validators: Array<Validator>;
     values: Array<string>;
-    additionalClassName?: string;
     validationLabelInfo?: string;
-    processNameValidationError?: string;
 };
 
 export default function ValidationLabels(props: Props) {
-    type ValidationErrors = {
+    type ValidationError = {
```

Comment:
also could/should be outsied of ValidationLabels

Reply:
done